composer.json

```diff
@@ -2,7 +2,15 @@
     "name": "pusher/pusher-push-notifications",
     "version": "0.10.5",
     "license": "MIT",
-    "require": {},
+    "require": {
+        "php": ">=5.6.0",
+        "guzzlehttp/guzzle": "~6.0"
```

Comment:
We are adding an extra dependency here, do we really benefit using this over plain curl?

Reply:
We are actually exchanging one dependency for another - we no longer require the PHP cURL extension!